pkg/account/delete/loader.go

```diff
@@ -126,16 +140,17 @@ func parseDeleteFileDefinition(definition FileDefinition) (Resources, error) {
 			case "environment":
 				environmentPolicies = append(environmentPolicies, EnvironmentPolicy{Name: parsed.Name, Environment: parsed.Level.Environment})
 			default:
-				return Resources{}, newDeleteEntryParserError(fmt.Sprintf("%v", e), i, fmt.Sprintf(`unkown policy level %q - needs to be one of "account","environment"`, parsed.Level))
+				return Resources{}, newDeleteEntryParserError(fmt.Sprintf("%v", e), i, fmt.Sprintf(`unknown policy level %q - needs to be either "account" or "environment"`, parsed.Level))
 			}
 		default:
-			return Resources{}, newDeleteEntryParserError(fmt.Sprintf("%v", e), i, fmt.Sprintf(`unkown type %q - needs to be one of "user","group","policy"`, parsed.Type))
+			return Resources{}, newDeleteEntryParserError(fmt.Sprintf("%v", e), i, fmt.Sprintf(`unknown type %q - needs to be one of "user", "service-user", "group", "policy"`, parsed.Type))
```

Comment:
We should not include `service-user` in here. On the one hand it's behind a feature flag and on the other hand we are always printing that service-user may not be set

Reply:
Good catch :)
I will remove `service-user` for the  moment, but maybe we remove the examples from the message - better to encourage people to read the docs.